packaging/linux/smoketest/smoketest_all.bash

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/bash
+set -euox pipefail
+echo "Starting linux smoketests"
+
+versionstring=$1
+version=$(echo "$versionstring" | cut -d'-' -f1)
+datetime=$(echo "$versionstring" | cut -d'-' -f2 | cut -d'+' -f1)
+revision=$(echo "$versionstring" | cut -d'+' -f2)
+curl --output /dev/null --silent --head --fail "https://s3.amazonaws.com/jack-testing.keybase.io/linux_binaries/deb/keybase_${version}-${datetime}.${revision}_amd64.deb"
```

Comment:
Can you add a comment explaining that these lines are here just so that the script fails early if our downloads are down?

Also do you really want `jack-testing`?

Reply:
>downloads are down
Or more likely, not built yet. E.g. you have to !test commitid to build the test build and then smoketest them.

Reply:
and will change jack-testing in a subsequent pr